.code-samples.meilisearch.yaml

```
@@ -4,74 +4,323 @@
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
 get_one_index_1: |-
+  $client->getIndex('movies');
 list_all_indexes_1: |-
+  $client->getAllIndexes();
 create_an_index_1: |-
+  $client->createIndex('movies', ['primaryKey' => 'movie_id']);
 update_an_index_1: |-
 delete_an_index_1: |-
+  $client->deleteIndex('movies');
 get_one_document_1: |-
+  $index = $client->getIndex('movies');
+  $index->getDocument(25684);
```

```suggestion
  $document = $client->getIndex('movies')->getDocument(25684);
```

I'm not sure if this is possible in PHP and if it's good practice. 
We try to avoid as much as possible multi-lines examples to keep the code examples clean and short. 

What do you think?

---

This is possible :smile:

I understand and approve your idea !

I have edited the whole doc to be align with your suggestion.
I have just removed `$document =` to be align with what's done for javascript and python :smile:  (if this is ok for you of course)

---

Yes absolutely ! Thanks a lot :)